doc/method/results.md

```diff
@@ -0,0 +1,342 @@
+
+# Results
+
+## Alignment
+
+The alignment procedure examines every source node.  The following is
+a breakdown of the disposition of source nodes, across all sources, as
+OTT is assembled.  Explanations of the categories follow.
+
+```
+     49  curated alignment
+    105  align to skeleton taxonomy
+
+  21584  disjoint divisions
+    172  disparate ranks
+  25800  by lineage
+   7653  overlapping membership
+    219  same division
+  83012  by name
+
+2325870  confirmed
```

Comment:
like all the others, it's described in the following text (as it says in the opening paragraph).  not sure how I can make that clearer?